android-lib/src/main/java/com/kinvey/android/Client.java

```diff
@@ -125,14 +126,15 @@
     protected Client(HttpTransport transport, HttpRequestInitializer httpRequestInitializer, String rootUrl,
                      String servicePath, JsonObjectParser objectParser,
                      KinveyClientRequestInitializer kinveyRequestInitializer, CredentialStore store,
-                     BackOffPolicy requestPolicy, Context context) {
+                     BackOffPolicy requestPolicy, byte[] encryptionKey, Context context) {
```

Comment:
Is this a breaking change for this constructor?

Reply:
Yes, because we need the encryptionKey in 'Client' to initialize RealmCacheManager during Client initialization process and initialize new RealmCacheManager after logout (because android-sdk is cleared after logout).